cmake/flash/CMakeLists.txt

```diff
@@ -14,6 +14,15 @@ set(RUNNER_ARGS_COMMON
   "--openocd-search=${OPENOCD_DEFAULT_PATH}"
   )
 
+# Extra debugging, if requested.
+set(FLASH_DEBUG_VERBOSE OFF CACHE BOOL "If set, print verbose flash/debug output.")
```

Comment:
The below is used elsewhere to solve the same problem.

    $<$<BOOL:${CMAKE_VERBOSE_MAKEFILE}>:--verbose>

I think it makes sense to re-use CMAKE_VERBOSE_MAKEFILE here,
because it is easier for a user if he only needs to learn the name of
one CMake variable to pass --verbose to underlying commands.

See zephyr/arch/x86/CMakeLists.txt

Reply:
Thanks; I wasn't aware of that. I'll fix it.